### PR TITLE
Migrated to th2 gradle plugin `0.0.8`

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
* Updated bom: `4.6.1`
* Updated common: `5.12.0-dev`
* Updated common-utils: `2.2.3-dev`
* Updated cradle api: `5.3.0-dev`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,127 +32,20 @@ repositories {
...
-                'Implementation-Version': project.version
-        )
-    }
-}
-
 configurations.configureEach {
     resolutionStrategy {
         force "com.exactpro.th2:cradle-core:$cradleVersion"
         force "com.exactpro.th2:cradle-cassandra:$cradleVersion"
     }
```
Comment (OptimumCode) [inline build.gradle:43]: thought: Maybe we should just exclude these dependencies from `common`'s transitive dependencies? Like we did in other projects?

**Review by OptimumCode — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:43]: Do you mean mark cradle dependency as compileOnly ?

Comment (OptimumCode) [reply build.gradle:43]: No, I meant this: https://github.com/th2-net/th2-rpt-data-provider/blob/master/build.gradle#L49

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:43]: Got it, thank you

**Review by OptimumCode — DISMISSED** [review]

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**